### DISCUSSION TITLE
Is there a "Monaco" of Lapce?

### DISCUSSION THREAD

**Discussion by FredrikNoren** [lapce/lapce#2780] [Q&A] [answered]
VSCode's core code editor [Monaco](https://github.com/microsoft/monaco-editor) is a standalone library that can be integrated into any web app. Is there an equivalent for Lapce? Specifically for integrating it into other wgpu applications.

**Comment by panekj** [comment, marked as answer]
No, Lapce editor is written specifically to be only usable by Lapce itself as it gives us better integration into the codebase

**Comment by MinusGix** [comment]
I'm working on extracting the editor into a separate crate, but it would be for use with Floem (our UI library) not arbitrary WGPU applications.
